meatnet_node_ble_specification.rst

```diff
@@ -1124,23 +1124,20 @@ sync purposes. The values are interpreted exclusively by the client in
 Integrated mode; the firmware performs the food safety calculations based on
 the other values supplied.
 
```

Comment:
Please add a note to these lists in both documents explaining that missing values are deprecated, to guide future maintainers.

Reply:
@jmaha added a note - what you expected?

Reply:
That works, thanks!